site/.storybook/preview.jsx

```diff
@@ -24,6 +25,13 @@ export const decorators = [
       </HistoryRouter>
     )
   },
+  (Story) => {
```

Comment:
why do we need this now?

Reply:
This is how the new Storybook version recommends using it when creating stories. https://storybook.js.org/docs/react/writing-stories/introduction#defining-stories